packages/gallery/src/components/gallery/leanGallery/leanGallery.js

```diff
@@ -94,18 +94,43 @@ export default class LeanGallery extends React.Component {
     const { styles } = this.props;
     const { gridStyle, numberOfImagesPerRow, imageMargin } = styles;
 
-    // const minmaxFix = 0.75; //this fix is meant to compensate for the css grid ability to use the number as a minimum only (the pro-gallery is trying to get as close as possible to this number)
-    const minmaxFix = 1;
+    const minmaxFix = 1; //this fix is meant to compensate for the css grid ability to use the number as a minimum only (the pro-gallery is trying to get as close as possible to this number)
```

Comment:
if it is 1, remove it.
We don't store old code as comments, github is our archive